docs/stellar-core_example.cfg

```diff
@@ -101,6 +101,11 @@ KNOWN_PEERS=[
 "core-testnet2.stellar.org", 
 "core-testnet3.stellar.org"]
 
+# BOOTSTRAP_DNS_SRV (single string) default is '_stellar-core._tcp.stellar.org'
```

Comment:
I get how that is kinda cool but I don't really think SDF should be the default for all that stuff. It is in the example config so it is very easy for people to use. It also leads to unexpected behavior when you do know what is going on. "I didn't put anything in your history store why is it still fetching history!?"